skins/material3/QskMaterial3SkinFactory.h

```diff
@@ -12,9 +12,10 @@
 class QSK_MATERIAL3_EXPORT QskMaterial3SkinFactory : public QskSkinFactory
 {
     Q_OBJECT
-
+#if defined( QSK_MATERIAL3_MAKEDLL )
```

Comment:
Otherwise while building as `STATIC` libraries in conjunction with `ENABLE_ENSURE_SKINS` enabled the following error occurs:

````bash
9>material3skin.lib(QskMaterial3SkinFactory.obj) : error LNK2005: qt_plugin_query_metadata already defined in squiekskin.lib(QskSquiekSkinFactory.obj)
9>material3skin.lib(QskMaterial3SkinFactory.obj) : error LNK2005: qt_plugin_instance already defined in squiekskin.lib(QskSquiekSkinFactory.obj)
9>   Creating library G:/repositories/qskinny-build/examples/iotdashboard/Debug/iotdashboard.lib and object G:/repositories/qskinny-build/examples/iotdashboard/Debug/iotdashboard.exp
9>G:\repositories\qskinny-build\examples\iotdashboard\Debug\iotdashboard.exe : fatal error LNK1169: one or more multiply defined symbols found
```